### DISCUSSION TITLE
Feature request - [Print "Speech" while talking]

### DISCUSSION THREAD

**Discussion by RoboEvangelist** [snakers4/silero-vad#548] [Q&A] [answered]
Is there a way to allow silero to print "possible speech detected" every N milliseconds? This is as opposed to waiting for a recording to finish and then printing "possible speech detected" at the end

**Comment by snakers4** [comment, marked as answer]
The VAD outputs probability each ~30ms. So you are free to do any post-processing / aggregation you would like.

A good start is to modify the logic behind the get speech timestamps function.
